website/content/docs/integrations/consul/acl.mdx

```diff
@@ -109,7 +109,10 @@ Nomad uses to sign workload identities. With these keys, Consul is able to
 validate their origin and confirm that they were actually created by Nomad.
 
 Nomad cannot recreate Consul tokens that have been deleted. The auth method
-configuration should never set the `MaxTokenTTL` field.
+configuration should never set the `MaxTokenTTL` field. Consul tokens will be
+local to the Consul datacenter unless you set `TokenLocality: "global"` in the
+auth method. We recommend using local tokens (the default) because global tokens
```

Comment:
```suggestion
auth method. We recommend using local tokens (the default) because global tokens
```
```suggestion
auth method. We recommend using local tokens, which is the default. Global tokens
```
From the Google style guide "Unfortunately, some readers ignore anything that appears in parentheses, so don't put important information in parentheses if you can help it."